Documentation/requirements.md

```diff
@@ -0,0 +1,29 @@
+# Requirements
+
+## Ports
+
+The information below describes a minimum set of port allocations used by Kubernetes components.
+
+### Master node(s) ingress
+
+| Protocol | Port Range | Source                                    | Purpose                |
+-----------|------------|-------------------------------------------|------------------------|
+| TCP      | 443        | Worker Nodes, API Requests, and End-Users | Kubernetes API server. |
+| UDP      | 8472       | Master & Worker Nodes                     | flannel overlay network - *vxlan backend* |
```

Comment:
TCP `10251` for kube-scheduler healthz
TCP `10252` for kube-controller-manager healthz